src/main/java/servlets/ReceiptAnalysisServlet.java

```diff
@@ -41,13 +40,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       return;
     }
 
-    try {
-      results = ReceiptAnalysis.analyzeImageAt(new URL(url));
-    } catch (ReceiptAnalysisException e) {
-      response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
```

Comment:
I suppose we could have just returned an empty object here instead of an error?

Reply:
The `ReceiptAnalysisServlet` is not used in the flow, so that would not affect the application.